tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -41,17 +41,29 @@ Verify Traefik Deployment
     ...       Tier1
     ...       ODS-546
     ...       ODS-552
+     ${version-check} =  Is RHODS Version Greater Or Equal Than  1.16.0
+    Skip if   ${version-check}==True
+    ...   Traefik proxy is removed after KFNBC migration
```

Comment:
Since this is going in the KFNBC branch, the version check is not needed. You can skip directly.
Same for all other version checks

Reply:
I think it is good idea to keep it once it become master

Reply:
I don't think it will be needed since we have a 1.15 branch that was created for the release, and master will become the branch for 1.16

Reply:
Done